### PR TITLE
Some polish due to WGLC feedback

### PR DESCRIPTION
**Pull request by henkbirkholz**
* some terminology
* a few synonyms
* restructuring of the acknowledgement section

Please feel free to bash.

### CODE REVIEW

File: draft-ietf-rats-endorsements.md [hunk 1]
```diff
@@ -371,7 +371,7 @@ trust in the Verifier it will use.  In such a case, the Relying Party may have
 a constrained Verifier embedded in it that is only capable of appraising Evidence
 provided by its desired Verifier.  Thus, the Relying Party uses its embedded Verifier
 for purposes of appraising its desired Verifier which it treats as only an Attester,
-and once verified, then uses it for verification of all other Attesters.
+and once verified via appraisal, then uses it for appraisal of all other Attesters.
```
Comment (thomas-fossati) [inline draft-ietf-rats-endorsements.md]: I find _"verified via appraisal"_ unnecessarily heavy.

Let's do _"appraised"_, or leave it as-is.

(The former seems more in line with the PR's spirit.)

**Review by thomas-fossati — APPROVED** [review]
LGTM, with one small comment.

File: draft-ietf-rats-endorsements.md (hunk 1 above)
Comment (henkbirkholz) [reply draft-ietf-rats-endorsements.md]: okay

Comment (henkbirkholz) [inline draft-ietf-rats-endorsements.md]:
```suggestion
and once appraised, then uses it for appraisal of all other Attesters.
```

**Review by dthaler — APPROVED** [review]

**Pull request merged**